### ISSUE TITLE
Forearm pain subtypes

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10785]
**Preferred term label:**
Dorsal forearm pain
Volar forearm pain

**Synonyms**
Pain localized to dorsal (non-palmar) aspect of the forearm
Pain on the volar (palmar) aspect of the forearm

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:6000299 (which is missing the word pain in the title)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dorsal:
Radial tunnel syndrome

Volar
Pronator syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001357
- label: Dorsal forearm pain
- parents: [Forearm pain (HP:6000299)](https://hpo.jax.org/app/browse/term/HP:6000299)
- pmid: 
- def: Pain localized to dorsal (non-palmar) aspect of the forearm.
- comment: Dorsal forearm pain can be observed with radial tunnel syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10785
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001358
- label: Volar forearm pain
- parents: [Forearm pain (HP:6000299)](https://hpo.jax.org/app/browse/term/HP:6000299)
- pmid: 
- def: Pain on the volar (palmar) aspect of the forearm.
- comment: Volar forearm pain can be observed with conditions such as pronator syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10785
- synonyms:
